modules/ossm-migrating-gateways-in-place.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-gateways-in-place_{context}"]
+= Gateway in place migration
+
+If you need less detailed control over your gateway migration, you can perform gateway in place migration. This method applies to namespaces with dedicated gateways, and to environments using a centralized gateway shared across multiple namespaces.
+
+[NOTE]
+====
+The label for the gateway namespace differs between mulitenant and cluster-wide meshes. To understand which labels to use in your specific migration case and the `maistra.io/ignore-namespace: "true"` label requirement, see the "Multitenant migration guide" or the "Cluster-wide migration guide" section.
+====
+.Procedure
+
+. Run the following command to label the gateway namespace to ensure that gateway injection is enabled from the new mesh and add the `maistra.io/ignore-namespace: "true"` label:
++
+[source,terminal]
+----
+$ oc -n <app_namespace> rollout restart deployment <gateway_name>
+----
++
+This command also removes the `istio-injection=enabled` label, if needed.
```

Comment:
This is not true. Where is the command above coming from? Probably some error in editing.